shared/src/msi-installer/Product.wxs

```diff
@@ -44,12 +44,6 @@
 
       <ComponentGroupRef Id="ContinuousProfiler.EnvironmentVariables.Machine"/>
       <ComponentGroupRef Id="ContinuousProfiler.Files"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.Net45.GAC"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.Net461.GAC"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.Net45"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.Net461"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.NetStandard20"/>
-      <ComponentGroupRef Id="ContinuousProfiler.Files.Managed.Netcoreapp31"/>
```

Comment:
(Add a comment here since I cannot add one farther in the file)
I do not know what to do with declarations at line [107](https://github.com/DataDog/dd-trace-dotnet/pull/2798/files#diff-9d5572b036422073a6c989a923b22036f50e43bc87bc5e690e337bb163264bb7L107). Any idea @zacharycmontoya

Reply:
If you no longer need the directories for the managed files, then you can just delete the related directory nodes. It will be safe to do so because you've already deleted all of the "components" that are referencing them